### DISCUSSION TITLE
cffi

### DISCUSSION THREAD

**Discussion by Houqua3D** [flet-dev/flet#3165] [Packages] [answered]
`cffi` is a C Foreign Function Interface for Python. Required by `cryptography` package.
https://pypi.org/project/cffi/#files

**Comment by FeodorFitsner** [comment, marked as answer]
[`cffi`](https://pypi.flet.dev/cffi) and [other packages](https://pypi.flet.dev/) are available in Flet 0.25.0 pre-release and up for both iOS and Android. [Give it a try](https://flet.dev/blog/flet-new-packaging-pre-release) and let us know how that works.
